lib/tasks/tmp_international_development_funds.rake

```diff
@@ -0,0 +1,17 @@
+namespace :international_development_funds do
+  desc "Republishes international development funds with new 'value_of_funding' values"
+  task update_value_of_funding: :environment do
+    Edition.where(document_type: "international_development_fund").each do |edition|
+      value_of_funding = edition.details.dig(:metadata, :value_of_funding) || []
+      next unless value_of_funding.include?("up-to-100000")
+
+      other_values = value_of_funding.reject { |val| val == "up-to-100000" }
+      edition.details = edition.details.deep_merge({
+        metadata: {
+          value_of_funding: other_values.unshift("10001-to-100000"),
```

Comment:
Not related to your PR, but `unshift` seems like a very poorly named function for what this actually does!

Reply:
I've never agreed with the `shift`/`unshift` naming of these methods 😁

Reply:
...I enjoyed [this StackOverflow answer](https://stackoverflow.com/questions/6085518/what-is-the-easiest-way-to-push-an-element-to-the-beginning-of-the-array#comment7055991_6085543) too 💩

Reply:
Haha that's amazing!